ssts/utils/seg_label_utils.py

```diff
@@ -0,0 +1,350 @@
+import os
```

Comment:
What are the methods in here ("seg_label_utils") used for?

Reply:
I've added a descriptive, intro doc-string to the module